app/controllers/aboutyou/ChildBenefitController.scala

```diff
@@ -46,8 +45,7 @@ class ChildBenefitController @Inject()(
 
   def form(isAgent: Boolean) = formProvider(isAgent)
 
-  def onPageLoad(mode: Mode, taxYear: Int): Action[AnyContent] =
-    (identify(taxYear) andThen taxYearAction(taxYear) andThen getData(taxYear) andThen requireData(taxYear)) {
+  def onPageLoad(mode: Mode, taxYear: Int): Action[AnyContent] = (identify(taxYear) andThen getData(taxYear) andThen requireData(taxYear)) {
```

Comment:
we need to add taxYear validation ->  `taxYearAction(taxYear) andThen` in all places